Updated Date and Time Functions List (#1)
Bit-Quill/documentation-website: _search-plugins/sql/functions.md

tan | `tan(number T) -> double` | `SELECT tan(0.5) FROM my-index LIMIT 1`

## Date and time

> Functions marked with * are not yet implemented in PPL, with plans to support them in the near future.

Reviewer: ```suggestion
Functions marked with * have future PPL implementation on roadmap.
```
